test/parallel/test-runner-coverage.js

```diff
@@ -25,15 +25,15 @@ function getTapCoverageFixtureReport() {
   /* eslint-disable @stylistic/js/max-len */
   const report = [
     '# start of coverage report',
-    '# -------------------------------------------------------------------------------------------------------------------',
+    '# ' + '-'.repeat(115),
```

Comment:
please revert, this is both unrelated and IMO does not improve readability

Reply:
It's not for readability, it was because it increased it had a line length of >120. I changed it in my changes, didn't realize this spot add an `eslint-disable` directive.

*resolved locally*